builder/dockerfile/instructions/parse.go

```diff
@@ -534,8 +534,11 @@ func parseVolume(req parseRequest) (*VolumeCommand, error) {
 
 	for _, v := range req.args {
 		v = strings.TrimSpace(v)
-		if v == "" {
+		switch {
+		case v == "":
 			return nil, errors.New("VOLUME specified can not be an empty string")
+		case v == "." || v == "/":
+			return nil, fmt.Errorf("VOLUME specified can not be '%s'", v)
```

Comment:
Isn't `.` valid as long as WORKDIR is used to change to a path other than `/` ?

Reply:
Right now, even when setting `WORKDIR` to `/tmp/mydir` and setting `VOLUME` to `.` I get a weird error: 
```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:296: starting container process caused "open /proc/self/fd: no such file or directory": unknown.
```

Reply:
Ah, I guess WORKDIR does not impact VOLUME, so this is correct. Thanks for confirming!